crates/bevy_math/src/primitives/dim3.rs

```diff
@@ -171,3 +205,47 @@ pub struct Capsule {
 }
 impl super::Primitive2d for Capsule {}
 impl Primitive3d for Capsule {}
+
+impl Capsule {
+    /// Create a new `Capsule` from a radius and length
+    pub fn new(radius: f32, length: f32) -> Self {
+        Self {
+            radius,
+            half_length: length / 2.0,
+        }
+    }
+}
+
+/// A cone primitive.
```

Comment:
As @NiseVoid mentioned, maybe we should mention where the origin of the cone should be so that different integrations have consistent representations?

If the options are the base and apex (tip), the base is more intuitive for me personally. I view cones like [Wikipedia's description](https://en.wikipedia.org/wiki/Cone):
> a three-dimensional geometric shape that tapers smoothly from a flat base (frequently, though not necessarily, circular) to a point called the apex or vertex.